develop/dev-guide-proxysql-integration.md

```diff
@@ -104,579 +655,785 @@ The production environment requires more steps than the test environment. To dep
+    ```
+    mysql_servers:
+    (
+        {
+            address="<tidb cloud dedicated cluster host>"
+            port=<tidb cloud dedicated cluster port>
+            hostgroup=0
+            max_connections=2000
+        }
+    )
+
+    mysql_users:
+    (
+        {
+            username = "<tidb cloud dedicated cluster username>"
+            password = "<tidb cloud dedicated cluster password>"
+            default_hostgroup = 0
+            max_connections = 1000
+            default_schema = "test"
+            active = 1
+            transaction_persistent = 1
+        }
+    )
+    ```
 
-```shell
-rm /var/lib/proxysql/proxysql.db
-```
+3. Restart ProxySQL:
 
-Alternatively, you can execute the `LOAD xxx FROM CONFIG` command to overwrite the current configuration.
+    ```bash
+    systemctl restart proxysql
+    ```
 
-The path of the configuration file is `/etc/proxysql.cnf`. To configure required configuration items in the preceding sections with the configuration file, the following takes `mysql_servers` and `mysql_users` as an example. To modify other items, refer to the `/etc/proxysql.cnf`.
+    After the restart, the SQLite database will be created automatically.
 
-```
-mysql_servers =
-(
-    {
-        address="127.0.0.1"
-        port=4000
-        hostgroup=0
-        max_connections=2000
-    }
-)
-
-mysql_users:
-(
-    {
-        username = "root"
-        password = ""
-        default_hostgroup = 0
-        max_connections = 1000
-        default_schema = "test"
-        active = 1
-        transaction_persistent = 1
-    }
-)
-```
+> **Warning:**
+>
+> Do not run ProxySQL with default credentials in production. Before starting the `proxysql` service, you can change the defaults in the `/etc/proxysql.cnf` file by changing the admin_credentials variable.
 
-To make the preceding modifications effective, use the `systemctl restart proxysql` command to restart ProxySQL. Then the SQLite database will be created automatically and the configuration file will not be ignored.
+## Typical scenarios
 
-### Other configuration items
+This section of the document discusses some of the benefits that we can leverage by integrating ProxySQL with TiDB Server, such as Loadbalancing and query routing with some examples.
 
-The preceding configuration items are required. For optional configuration items, refer to [Global variables](https://proxysql.com/documentation/global-variables/).
+For these examples, we will be using the docker version of TiDB and ProxySQL to configure load balancing, so if you have been following along, the assumption is that you already have a setup installed and running, if not, visit our [getting started section](#option-2---integrate-tidb-self-hosted-with-proxysql) to learn more.
 
-## 4. Try out
+### Load balancing
 
```

Comment:
@Icemap add a NOTE Here

This scenario is more appropriate for self-hosted deployments of TiDB where scalability could be a challenge. TiDB Cloud and TiDB Serverless can process and scale with large number of requests eliminating this type of concern.

Reply:
@zhangyangyu fyi.